### PR TITLE
Torch 0.5 update.

### PR DESCRIPTION
**Pull request by LaurentMazare**
Update ocaml-torch to the latest version.

### CODE REVIEW

File: packages/torch/torch.0.5/opam [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+bug-reports:  "https://github.com/LaurentMazare/ocaml-torch/issues"
+homepage:     "https://github.com/LaurentMazare/ocaml-torch"
+dev-repo:     "git+https://github.com/LaurentMazare/ocaml-torch.git"
+maintainer:   "Laurent Mazare <lmazare@gmail.com>"
+authors:      [ "Laurent Mazare" ]
+
+build: [["dune" "build" "-p" name "-j" jobs]]
+
+depends: [
+  "base" {>= "0.11.0" & < "v0.13"}
```
Comment (kit-ty-kate) [inline packages/torch/torch.0.5/opam]:
```suggestion
  "base" {>= "v0.11.0" & < "v0.13"}
```
The `v` prefix is relevant

**Comment by kit-ty-kate** [comment]
Could you return this fix to the upstream opam file?

**Comment by kit-ty-kate** [comment]
Seems like the libtorch archive changed:
```
[ERROR] The sources of the following couldn't be obtained, aborting:
          - libtorch.1.1.0:
            https://download.pytorch.org/libtorch/cpu/libtorch-macos-1.1.0.zip (Bad checksum, expected md5=70c5046a83e4b8b33772bc969562da76)
```

**Comment by LaurentMazare** [comment]
> Could you return this fix to the upstream opam file?

Sure I just push a commit upstream for this, thanks for fixing this.
I've also updated the libtorch package to fix the md5 issue.

**Comment by mseri** [comment]
Thanks

**Pull request merged**